README.md

```diff
@@ -188,27 +186,31 @@ After doing all of those steps, we are ready! Double click the `Pal.uproject` fi
 > [!NOTE]
 > You may think nothing is happening, unreal engine has to compile a lot of things in the background before the editor opens. This may take a LONG while.
 
-
 ## What do I do now?
 
 > [!IMPORTANT]
 > The next step in the process will be to create a Compatible Blueprint Mod
 
-- [UE4SS Github](https://github.com/UE4SS-RE/RE-UE4SS)
-- [UE4SS Offical Docs](https://docs.ue4ss.com/index.html)
-- [Creating A Blueprint Mod Youtube Video](https://www.youtube.com/watch?v=fB3yT85XhVA)  
-
+* [UE4SS Github](https://github.com/UE4SS-RE/RE-UE4SS)
+* [UE4SS Offical Docs](https://docs.ue4ss.com/index.html)
+* [Creating A Blueprint Mod Youtube Video](https://www.youtube.com/watch?v=fB3yT85XhVA)  
 
 ## Packaging
 
 > [!NOTE]
-> Make Sure These Two Settings Have Been Enabled ![EditorPrefExp](assets/EditorPrefrencesExperimental.png) ![ProjectSettingsPackaging](assets/ProjectSettingsPackaging.png)
-
-1) Create a Data Asset ( Primary Asset Label ) named after your mod name in your Contents folder. Set the priority to 1, and the Chunk ID to a memorable ID. 
```

Comment:
I've actually had chunk assignment options not show up for me sometimes, so it might be useful to include them as an optional note/warning